web/themes/custom/pineapple/sass/components/paragraphs/_image-paragraph.scss

```diff
@@ -1,8 +1,19 @@
 .paragraph--type--image-paragraph {
-  .image-paragraph-image {
-    img {
-      object-fit: cover;
-      width: 100%;
+  .image-paragraph-container {
+
+    .paragraph-image {
+      width: 680px;
+      height: 350px;
+      display: contents;
```

Comment:
This is not the correct way to go about this, you shouldn't be setting a set pixel value on images, they should take up the size of the container.

Reply:
okay, done